src/styles/base/_typography.scss

```diff
@@ -137,10 +137,16 @@ a {
     text-decoration: none;
   }
 
-  &:hover {
+  &:hover, &:focus, &.underline {
     text-decoration: underline;
   }
 
+  &.underline {
+    &:hover, &:focus {
+      text-decoration-thickness: 2px;
```

Comment:
I tested this and the change in thickness is a little jarring to me. 
I looked in the accessibility guide and can't seem to find information on how to handle hover state of a link that's already underlined. But I found that [mdn docs ]( https://developer.mozilla.org/en-US/docs/Learn/CSS/First_steps)disable the underline on hover, which is pretty interesting and maybe worth trying?